docs/modeling-your-data/running-data-models-via-snowplow-bdp/standard-models/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Standard models"
+sidebar_position: 2
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+### Overview
+Standard data models are authored and maintained by Snowplow. Follow below steps to create one.
+
+### 1. Create a warehouse connection
+Begin by creating a new warehouse connection. It will be used by a data model to connect to your warehouse. To create a new warehouse connection, click on the "Setup new target" button on the Data Models page and fill in all the necessary details.
+
+![](images/warehouse-connections.png)
+
+Currently, only Snowflake connections can be created. However, support for other warehouses will be added soon.
+
+### 2. Create a data model
+To create a new data model, click the "Add data model" button. Then, select the data model you're interested in, such as Unified Digital, and the warehouse connection it should use. You can also specify the owners of the data model who will be alerted in case of failure.
```

Comment:
```suggestion
To create a new data model, click the "Add data model" button. Then, select the data model you're interested in, such as [Unified Digital](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/index.md), and the warehouse connection it should use. You can also specify the owners of the data model who will be alerted in case of failure.
```